core/src/main/java/tech/pegasys/web3signer/core/multikey/SignerLoader.java

```diff
@@ -41,33 +46,62 @@
 
     LOG.info("Loading signer configuration files from {}", configsDirectory);
     final AtomicLong configFilesHandled = new AtomicLong();
+
+    final List<Path> configFilesList = getConfigFilesList(configsDirectory);
```

Comment:
:/ don't love getting the filenames from disk, then iterating over them - but can see that the log info is potentially useful.

Reply:
this allows to parallel-process files ... as Files.list.stream does not return parallel stream.